frontend/src/components/Common/LabeledDropdown/index.tsx

```diff
@@ -0,0 +1,55 @@
+import { DropdownProps, Label, Dropdown } from 'semantic-ui-react';
+import styles from './LabeledDropdown.module.css';
+
+interface Data {
+  key: string;
+  label: React.ReactNode;
+  info?: React.ReactNode[];
+}
+
+export interface LabeledDropdownProps extends DropdownProps {
+  data: Data[];
+}
+
+const Item = ({ label, info = [] }: Data): JSX.Element => (
+  <div style={{ display: 'flex', alignItems: 'center', minHeight: 26 }}>
+    <div style={{ flex: 1 }}>{label}</div>
+    {info
+      .filter((x) => x !== undefined)
+      .map((x, i) => (
+        <Label key={i} content={x} />
+      ))}
+  </div>
+);
+
+const LabeledDropdown = ({
```

Comment:
i really do like this work that you've done with the dropdowns! @team I was wondering if we've ever considered changing the way we account for extensions, instead of being in the dropdown, like keep the dropdown as 1,2,3,4,5  and then do like a extensions check box in the side?

Reply:
We can ask designers about this. It might also involve a backend change because "extensions" are just a name we gave the assignments (e.g., "portfolio #1" and "portfolio #1 extension").

Reply:
I like the idea of having an extensions checkbox on the side, but the leads don't always offer extensions for every portfolio, so this may not be necessary.